tests/FsYacc.Core.Tests/Sample.fs

```diff
@@ -0,0 +1,9 @@
+﻿module FsYacc.Core.Tests.Sample
+
+open Expecto
+
+[<Tests>]
+let tests =
+    test "sample" {
```

Comment:
I had trouble with quickly coming up with a meaningful test for the fsyacc part so I decided to go with a sample to leave it "ready" for real tests in the future.